scripts/ci/linkRepos.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+# Description: links any injected tap in the keg-hub/repos directory. Ignores the others.
+
+source $KEG_CLI_PATH/keg
+
+log_result () {
+  if [[ $2 -eq 1 ]]; then
+    echo "No alias key in tap config for \"$1\". Skipping..."
+  fi
+}
+
+link_repos() {
+  local KEG_REPOS=$(ls "$KEG_HUB_PATH/repos")
+  for REPO in $KEG_REPOS
```

Comment:
Personally I like keeping the `do` part of the `for` loop on the same line
Not blocking, but for me it makes sense that they go together =>
```sh
for REPO in $KEG_REPOS; do
  # ...code stuff
done
```

Reply:
sure, updated

Reply:
For some reason, you're most recent commit is not showing up. but I do see the change in the commit directly. Odd